readthedocs/templates/projects/environmentvariable_detail.html

```diff
@@ -0,0 +1,23 @@
+{% extends "projects/project_edit_base.html" %}
+
+{% load i18n %}
+
+{% block title %}{% trans "Environment Variables" %}{% endblock %}
+
+{% block nav-dashboard %} class="active"{% endblock %}
+
+{% block editing-option-edit-environment-variables %}class="active"{% endblock %}
+
+{% block project-environment-variables-active %}active{% endblock %}
+{% block project_edit_content_header %}
+  {% blocktrans trimmed with name=environmentvariable.name %}
+    Environment Variable: {{ name }}
+  {% endblocktrans %}
+{% endblock %}
+
```

Comment:
I think we need copy here saying "The value here is not shown for security purposes" or similar. Otherwise it's a weird UI.